routers/router.ewr1.yml

```
@@ -55,3 +55,16 @@
     remote_address: us-nyc1.dn42.sherpherd.top
     remote_port: 20207
     public_key: PXkGrHooCFnB6N8bD6isgVjhl9GvzD2CLqlA+e2XikM=
+
+  name: STARSTORM-NYC1
```

```suggestion
- name: STARSTORM-NYC1
```

This needs to be the start of the YAML list containing your peering info.

You can look at the other peers in this file as an example.